usl_models/tests/flood_ml/test_dataset.py

```diff
@@ -86,45 +86,48 @@ def mock_blob_func(blob_name):
     # geospatial will have the labels creeping into a sequence of zeros.
     numpy.testing.assert_array_almost_equal(
         element["spatiotemporal"].numpy(),
-        numpy.array(
-            [
-                numpy.array(
-                    [
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                    ]
-                ),
-                numpy.array(
-                    [
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        mock_labels[:, :, 0],
-                    ]
-                ),
-                numpy.array(
-                    [
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        mock_labels[:, :, 0],
-                        mock_labels[:, :, 1],
-                    ]
-                ),
-                numpy.array(
-                    [
-                        numpy.zeros((1000, 1000)),
-                        numpy.zeros((1000, 1000)),
-                        mock_labels[:, :, 0],
-                        mock_labels[:, :, 1],
-                        mock_labels[:, :, 2],
-                    ]
-                ),
-            ]
+        numpy.expand_dims(
```

Comment:
Can you write out the tensor we're expecting rather than call `expand_dims` here? It's hard to tell what we're testing if the test code mirrors the calls in the code being tested to produce the expected outputs.

Reply:
Done.